ethcore/res/ethereum/classic.json

```diff
@@ -44,8 +44,8 @@
 		"eip1283Transition": "0xa03ae7",
```

Comment:
```suggestion
		"eip1283Transition": "0x0",
		"eip1283DisableTransition": "0x0",
```

Reply:
this is technically correct, but logically not. EIP-1283 was never activated on block `0`.

Reply:
that's right. this is just how foundation.json does it. don't know why.